app/models/user_note.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: user_notes
+#
+#  id                :bigint(8)        not null, primary key
+#  account_id        :bigint(8)
+#  target_account_id :bigint(8)
+#  comment           :text             not null
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#
+class UserNote < ApplicationRecord
```

Comment:
In my mind, it was more “note set by an user”, but unsure of how much sense it makes. It was also to lower the confusion that using “Note” could cause since it's ActivityPub vocabulary